tests/random_test.py

```diff
@@ -1119,6 +1119,12 @@ class A: pass
     with self.assertRaisesRegex(TypeError, 'unrecognized type .* PRNG'):
       jax.random.key(42, impl=A())
 
+  @jtu.sample_product(name=['threefry2x32', 'rbg', 'unsafe_rbg'])
```

Comment:
```suggestion
  @jtu.sample_product(name=list(dict(PRNG_IMPLS).keys()))
```
... or something along these lines? So as not to hard-code the names.

Reply:
I started with that, but I thought hard-coding was clearer, because it took me a handful of jumps throught he code to figure out what the values were 😁 

I can switch it back if you prefer.

Reply:
although `PRNG_IMPLS` is a list of tuples rather than a dict, so it would have to be something like
```python
name = [name for name, _ in PRNG_IMPLS]
```

Reply:
OK, sounds good. Keep as is then.

Reply:
OK, done